vendor/github.com/tidepool-org/clinic/client/client.go

```diff
@@ -4615,9 +4615,9 @@ func NewListPatientsRequest(server string, clinicId ClinicId, params *ListPatien
 
 		}
 
-		if params.CgmLastUploadDateFrom != nil {
```

Comment:
@Roukoswarf why was this deleted? Won't this cause issues for other clients (Tidepool Web)?

Reply:
this was a change from a previous ticket, clinic client has not been updated here in a while. Blip has already been switched to the new value.